action.yml

```
@@ -20,6 +20,10 @@ inputs:
     description: "The path for generated output"
     required: true
     default: "./.build/documentation"
+  minimum-access-level:
+    description: "The minimum access level of the symbols which should be included (public, internal or private)"
```

Nit: [Oxford comma](https://en.wikipedia.org/wiki/Serial_comma).

```suggestion
    description: "The minimum access level of the symbols which should be included (public, internal, or private)"
```